### PR TITLE
96b_carbon: Updated so that it compiles in the current codebase.

### PR DESCRIPTION
**Pull request by esden**
The pull request we merged to add 96b carbon target was a tiny bit stale. It needed some TLC :D

### CODE REVIEW

File: src/platforms/96b_carbon/platform.h [hunk 1]
```diff
@@ -100,9 +97,9 @@
  * For now USART preempts USB which may spin while buffer is drained.
  * TIM3 is used for traceswo capture and must be highest priority.
  */
-#define IRQ_PRI_USB		(2 << 4)
-#define IRQ_PRI_USBUSART	(1 << 4)
-#define IRQ_PRI_USBUSART_TIM	(3 << 4)
+#define IRQ_PRI_USB		(1 << 4)
```
Comment (dragonmux) [inline src/platforms/96b_carbon/platform.h:100]: Could you update the comment above as well so it reflects the new prioritisations?

**Review by dragonmux — CHANGES_REQUESTED** [review]
Looks decent, we have one thing that would be good to see changed though

File: src/platforms/96b_carbon/platform.h (hunk 1 above)
Comment (esden) [reply src/platforms/96b_carbon/platform.h:101]: Good catch. Thank you. I have updated the comment. Let me know if it is correct/sufficient. :)

**Review by dragonmux — APPROVED** [review]
LGTM :) ready to merge if you think it's complete

**Comment by esden** [comment]
Wonderful. Thanks! :)

**Pull request merged**